### PR TITLE
feat: Banner, FAQ UI revamp

### PR DESCRIPTION
**Pull request by thechefpenguin**
> [!NOTE]
> Some parts are commented as Ads won't be shown on all pages in the first phase. Will be iterated on in upcoming phase.

---

## PR-Codex overview
This PR introduces various enhancements and new components for the `AdPanel`, improves the layout and structure of pages, and integrates new features such as expandable ads and a countdown component. It also refines existing components for better usability and responsiveness.

### Detailed summary
- Added `AdPanel` components: `AdButton`, `AdPlayer`, `AdCard`, etc.
- Introduced `ExpandableAd` and `Countdown` components.
- Created `useShowAdPanel` and `useIsSlideExpanded` hooks.
- Updated `Page` to use `PageWithoutFAQ` layout in multiple pages.
- Enhanced `FAQ` configuration and rendering.
- Improved styling and responsiveness across components.
- Integrated new ad configurations for better rendering logic.

> The following files were skipped due to too many changes: `apps/web/src/components/AdPanel/Card.tsx`, `apps/web/src/components/AdPanel/CardLayouts.tsx`, `apps/web/src/views/Swap/Twap/TwapSwap.tsx`, `packages/localization/src/config/translations.json`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/components/AdPanel/Variations/TitleContentAd.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { AdButton } from '../Button'
+import { AdCard } from '../Card'
+import { Content } from '../Content'
+import { Title } from '../Title'
+
+export const TitleContentAd = () => {
+  return (
+    <AdCard imageUrl="/images/adpanel-test/bannerImg1.png">
+      <Title>Need Help?</Title>
```
[the remark this answers is not in the document]
Comment (thechefpenguin) [reply apps/web/src/components/AdPanel/Variations/TitleContentAd.tsx:9]: It has just begun development, this Draft PR is to keep track until its final. I will put a message in PR description stating this, thanks for starting to review

**Comment by ChefMomota** [comment]
Remember to remove all the assets image

File: apps/web/src/components/AdPanel/Variations/ContentAd.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { BodyText } from '../BodyText'
+import { AdButton } from '../Button'
+import { AdCard } from '../Card'
+import { AdPlayerProps } from '../types'
+import { getImageUrl } from '../utils'
+
+export const ContentAd = (props: AdPlayerProps) => {
+  return (
+    <AdCard imageUrl={getImageUrl('ad_clamm_options_trading.png')} {...props}>
+      <BodyText fontSize="15px">Web3 Notifications Now Live!!</BodyText>
```
Comment (chef-ryan) [inline apps/web/src/components/AdPanel/Variations/ContentAd.tsx:10]: Should add translation here?

Comment (thechefpenguin) [reply apps/web/src/components/AdPanel/Variations/ContentAd.tsx]: Components in `Variations` are for reference. Have added translations 👍🏻

**Review by Chef-Yogi — APPROVED** [review]

**Pull request merged**